client/src/components/Dropdown.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable */
+import axios from 'axios';
+import React, { useEffect, useState, useRef }  from 'react';
+// import genrePicture from '../BookGenreDropdown.png';
+
+
+// we are going to utilize two api calls to get the books by genre then grab their title page
+
+function Dropdown() {
+
+  const dropdownRef = useRef(null);
+  const [isActive, setIsActive] = useState(false);
+  const [genre, setGenre] = useState('');
+  const onClickActive = () => setIsActive(!isActive);
+  const onClickHorror = () => setGenre('horror') ;
```

Comment:
keep working on functionality but think about how you can make this onClick block DRY